gradio/interface.py

```diff
@@ -171,12 +171,20 @@ def __init__(self, fn, inputs=None, outputs=None, verbose=False, examples=None,
         self.server_port = server_port
         self.simple_server = None
         self.allow_screenshot = allow_screenshot
-        self.allow_flagging = os.getenv("GRADIO_FLAGGING") or allow_flagging
+        # If parameter is provided, use that; otherwise, environmet variable; otherwise, True
```

Comment:
Remove comment, self evident through code. Also, shouldn't allow_flagging be default set to None?

Reply:
Sure